### PR TITLE
Add Profile Creation Page1

### PR DESCRIPTION
**Pull request by Etesam913**
It's not completely finished (still some bugs left), however, I still think it would be valuable to get a review. Adds the tags, inputs, checkboxes, and gray tags for the page. 

This is how it looks:
Screen Shot 2020-11-15 at 2 20 54 PM

### CODE REVIEW

File: client/src/index.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@700&family=Roboto:wght@500&display=swap');
```
Comment (erin2722) [inline client/src/index.css:1]: maybe leave in roboto for now to give everyone a chance to switch?

File: src/containers/Test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (erin2722) [inline src/containers/Test.js:1]: remove this file before merging?

File: src/components/websiteTitle/index.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```
Comment (erin2722) [inline src/components/websiteTitle/index.js:1]: there seems to be two of these files, remove one?

File: client/src/containers/ProfileCreation/ProfileCreation1/index.js [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+        <MeetFrequency
+          clubProfile={clubProfile}
+          setClubProfile={setClubProfile}
+        />
+      </Column>
+    </StyledBody>
+  );
+};
+
+export default ProfileCreation1;
```
Comment (erin2722) [inline client/src/containers/ProfileCreation/ProfileCreation1/index.js:85]: maybe separate this and the below code into different files? this gets pretty long

Comment (erin2722) [reply client/src/containers/ProfileCreation/ProfileCreation1/index.js:85]: but keep all file within the ProfileCreation Container?

File: client/src/containers/ProfileCreation/ProfileCreationMaster.js [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+
+  const nextPage1 = () => {
+    let tempProfile = { ...clubProfile };
+    tempProfile.clubName = clubNameInput.current.value;
+    tempProfile.shortDesc = shortDescInput.current.value;
+    tempProfile.longDesc = longDescInput.current.value;
+
+    // Left empty
+    if (
+      tempProfile.clubName === '' ||
```
Comment (erin2722) [inline client/src/containers/ProfileCreation/ProfileCreationMaster.js:44]: alt shorter syntax:
```Javascript
if([tempProfile.clubName, 
tempProfile.shortDesc, 
tempProfile.longDesc, 
tempProfile.size, 
tempProfile.requireApplication].includes('') 
|| [tempProfile.memberPeriod, 
tempProfile.tags].includes([]));
```

File: client/src/containers/ProfileCreation/ProfileCreationMaster.js [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+      <Navbar />
+      <PageContainer>
+        <SetUpClubProfile />
+        {currentPath === '/profile-creation/1' && (
+          <PageDesc>Get-to-Know-You's: Why Students Should Join</PageDesc>
+        )}
+        {currentPath === '/profile-creation' && (
+          <PageDesc>Opening Statement: Provide Basic Info</PageDesc>
+        )}
+        <Router>
```
Comment (erin2722) [inline client/src/containers/ProfileCreation/ProfileCreationMaster.js:77]: we may want to document how this works a bit-- could be confusing for people coming into the codebase

**Review by erin2722 — CHANGES_REQUESTED** [review]
this looks really good 🎉 

I left a few comments, but everything is pretty much good to go 👍 

Check out the page on an iPhoneX view width-- the accepting applications buttons look a bit off.

This code is definitely on the more complex side, and so we may want to write some docs to explain it.

File: client/src/components/checkbox/index.js [hunk 7]
```diff
@@ -1,16 +1,31 @@
...
-  const onKeypress = e => {
+  const checkVariants = {
+    checked: { pathLength: 1 },
+    unchecked: { pathLength: 0 },
+  };
+
+  const onKeypress = (e) => {
     if (e.keyCode === 13) {
-      checkbox.current.click()
+      checkbox.current.click();
```
[the remark this answers is not in the document]
Comment (Etesam913) [reply client/src/components/checkbox/index.js:28]: Fixed. Problem had to do with which component my ref was on

File: client/src/containers/ProfileCreation/ProfileCreationMaster.js (hunk 6 above)
Comment (Etesam913) [reply client/src/containers/ProfileCreation/ProfileCreationMaster.js:77]: Added a comment to add some clarity.

**Review by erin2722 — APPROVED** [review]
🎉 This looks incredible! 

Ready to merge whenever you are.

**Pull request merged**